selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -145,6 +145,9 @@ checkParameters() {
         elif [[ "$var" == "--compare-with-ci" ]]; then :
         elif [[ "$var" =~ ^--workspace-pool-size=(auto|[0-9]+)$ ]]; then :
         elif [[ "$var" =~ ^[0-9]+$ ]] && [[ $@ =~ --compare-with-ci[[:space:]]$var ]]; then :
+        elif [[ "$var" =~ ^-D.* ]]; then :
+        elif [[ "$var" =~ ^-[[:alpha:]]$ ]]; then :
+        elif [[ "$var" == "--skip-sources-validation" ]]; then :
```

Comment:
What do you want to skip?

Reply:
Enforce plugin and source validation plugins.
This option add the following system properties:
```
-Dskip-enforce -Dskip-validate-sources
```

Reply:
I see. what for?

Reply:
For development purpose. It reduces time needed to run a selenium test by 40s.